src/js/components/Card.js

```diff
@@ -162,9 +230,17 @@ Card.propTypes = {
     PropTypes.element
   ]),
   reverse: PropTypes.bool,
+  transparent: PropTypes.bool,
+  link: PropTypes.string,
+  linkText: PropTypes.string,
+  linkIcon: PropTypes.element,
+  onClick: PropTypes.func,
   ...Tile.propTypes
```

Comment:
I'm wondering if Card should extend Box instead of Tile. What is it about Tile that makes it a good choice?